consensus/consensus.go

```diff
@@ -153,8 +153,9 @@ func (cs *consensusBase) Propose(cert hotstuff.SyncInfo) {
 	}
 
 	cs.blockChain.Store(proposal.Block)
-
-	cs.configuration.Propose(proposal)
+	if cs.kauri == nil {
+		cs.configuration.Propose(proposal)
```

Comment:
Would it be useful to have a comment explaining that Kauri won't propose here, but elsewhere?